study.md

```diff
@@ -31,7 +31,7 @@ In your own words, define what the responsibilities of the model layer are in
 Rails.
 
 ```md
-<!-- your answer here -->
+The model is all about your data. Any sort of create, update, destroy, or find methods should be here
```

Comment:
You model handles the data itself. It connects with the DB. The method actions are actually in the controller.